Basics/Calexp_guided_tour.ipynb

```diff
@@ -0,0 +1,754 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The WCS object can be used e.g. to convert pixel coordinates into sky coordinates"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wcs.pixelToSky(100.0, 100.0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's try accessing the metadata, and see what (header) keywords we have."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "metadata = calexp.getMetadata()\n",
+    "# help(metadata)\n",
+    "metadata.getOrderedNames()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "metadata.get('CCDTEMP')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> Post release 16.0, the `getName` method will be available."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Check if our calexp has a PSF"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "calexp.hasPsf()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "psf = calexp.getPsf()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The PSF object can be used to get a realization of a PSF at a specific point"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from lsst.geom.coordinates import Point2D\n",
+    "psfimage = psf.computeImage(Point2D(100.,100.))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Access the calibration object which can be used to convert instrumental magnitudes to AB magnitudes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "calib = calexp.getCalib()\n",
+    "calib"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Image cutouts"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can make a cutout from the calexp in our session."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import lsst.afw.geom as afwGeom\n",
+    "import lsst.afw.image as afwImage"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bbox = afwGeom.Box2I()\n",
+    "bbox.include(afwGeom.Point2I(2200,3200))\n",
+    "bbox.include(afwGeom.Point2I(2800,3800))\n",
+    "cutout = calexp.Factory(calexp, bbox, afwImage.LOCAL)"
```

Comment:
Much preferred these days:

    cutout = calexp[bbox, afwImage.LOCAL]

You can drop the `LOCAL` because you're not dealing with a subimage, making this even more cleaner:

    cutout = calexp[bbox]